pkg/volume/aws_ebs/attacher.go

```diff
@@ -231,7 +232,7 @@ func (detacher *awsElasticBlockStoreDetacher) WaitForDetach(devicePath string, t
 
 func (detacher *awsElasticBlockStoreDetacher) UnmountDevice(deviceMountPath string) error {
 	volume := path.Base(deviceMountPath)
-	if err := unmountPDAndRemoveGlobalPath(deviceMountPath, detacher.mounter); err != nil {
+	if err := volumeutil.UnmountPath(deviceMountPath, detacher.mounter); err != nil {
```

Comment:
The err is passed back to reconciler which also print the error log. Do we need to log it here to just return like gce_pd.go